teachopencadd/talktorials/T038_protein_ligand_interaction_prediction/talktorial.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.            with open(os.path.join(self.folder_name, "tables", "ligands.tsv"), "r") as data:
</pre><p>use path library for consistency</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>